test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ModelExpressionTagHelper.cshtml

```diff
@@ -1,6 +1,6 @@
-@model DateTime
+@model DateTime?
```

Comment:
Our generated code tried to set the model = null to start, since `DateTime` isn't nullable in order to keep this symantic we need the "?".